crates/anstyle-roff/tests/test_roff.rs

```diff
@@ -0,0 +1,59 @@
+use anstyle::{Color, RgbColor, Style, AnsiColor, XTermColor};
+
+#[test]
+fn test_rgb_color_output() {
+    let expected = include_str!("./roffs/rgb_color.roff");
+    let style = Style::new()
+        .fg_color(Some(Color::Rgb(RgbColor(255, 0, 170))))
+        .bg_color(Some(Color::Rgb(RgbColor(0, 255, 0))));
+
+    let mut roff_style = anstyle_roff::to_roff(style);
+    roff_style.text("test".to_owned());
+    assert_eq!(roff_style.render(), expected)
```

Comment:
Instead of `include_str!` the expected result, you can use `snapbox::assert_eq_path`.  This will make it easier to update everything as we evolve the code/tests.

Reply:
Done